crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/prosecution_concluded/validator/ProsecutionConcludedValidator.java

```diff
@@ -22,12 +39,37 @@ public void validateRequestObject(ProsecutionConcluded prosecutionConcluded) {
             throw new ValidationException(PAYLOAD_IS_NOT_AVAILABLE_OR_NULL);
     }
 
-    public Optional<Void> validateOuCode(String ouCode) {
-
-        if (!isBlank(ouCode)) {
-            return Optional.empty();
-        } else {
+    public String validateOuCode(String ouCode) {
+        if (isBlank(ouCode)) {
             throw new ValidationException(OU_CODE_IS_MISSING);
         }
+
+        try {
+            return crownCourtCodeHelper.getCode(ouCode);
+        }
+        catch (MAATCourtDataException ex) {
```

Comment:
I'd consider just changing the `CrownCourtCodeHelper` to throw a `ValidationException`

Reply:
Thanks, done